rts/at/sra/pom.xml

```diff
@@ -0,0 +1,184 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>org.jboss.narayana.quickstart.rts</groupId>
+        <artifactId>restat-quickstarts</artifactId>
+        <version>6.0.1.Final-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>sra-participant</artifactId>
+    <name>SRA Participant</name>
+    <packaging>jar</packaging>
+
+    <properties>
+        <narayana.version>${project.version}</narayana.version>
+        <!-- jakarta.servlet-api version 6.0.0 does not include jakarta/servlet/SingleThreadModel needed for undertow-->
+        <version.jakarta.servlet.jakarta-servlet-api>5.0.0</version.jakarta.servlet.jakarta-servlet-api>
```

Comment:
Actually it might have been me, I only had an issue with the agroal quickstart and I added a commit to the agroal repo to fix that so we should be able to use the boms now.